build/ci-build.yml

```diff
@@ -66,15 +67,16 @@ stages:
         pool:
           vmImage: 'ubuntu-16.04'
         steps:
-          - task: DownloadPipelineArtifact@2
-            displayName: 'Download build artifacts'
+          - task: UseDotNet@2
```

Comment:
'Use' doesn't overwrite previously installed SDK's. We may want to change something to the build as well; if we want both of them installed when compiling.

Reply:
LGTM